### PR TITLE
Update RELEASE_NOTES.md for 12.15 Release.

### PR DESCRIPTION
**Pull request by rhass**
Please feel free to add any missing release notes by committing to this branch

/cc @jeremymv2

### CODE REVIEW

**Comment by tas50** [comment]
We should note the changes to deleting registry entries

**Comment by tas50** [comment]
Also the sudo ssh change and the fix for the yum_repository regression

**Comment by jeremymv2** [comment]
Added some more info around exit code 213

**Comment by lamont-granquist** [comment]
CHANGELOG.md needs an update now, but i blew out my rate limit doing it wrong.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-## [v12.15.18](https://github.com/chef/chef/tree/v12.15.18) (2016-10-07)
-[Full Changelog](https://github.com/chef/chef/compare/v12.14.89...v12.15.18)
+## [v12.15.19](https://github.com/chef/chef/tree/v12.15.19) (2016-10-07)
+[Full Changelog](https://github.com/chef/chef/compare/v12.14.89...v12.15.19)
 
 **Enhancements:**
 
-- Adding support for rfc 62 exit code 213 [\#5428](https://github.com/chef/chef/pull/5428) ([jeremymv2](https://github.com/jeremymv2))
+- Adding support for rfc 62 exit code 213 (Chef upgrades) [\#5428](https://github.com/chef/chef/pull/5428) ([jeremymv2](https://github.com/jeremymv2))
```
Comment (lamont-granquist) [inline CHANGELOG.md]: this'll get blown away when anyone regenerates the changelog -- it needs to go into the title of the PR

Comment (rhass) [inline CHANGELOG.md]: I'll cherry-pick the changelog notes into a different PR and land them on master today.

**Pull request merged**